src/CircleBadge.js

```diff
@@ -2,27 +2,25 @@ import PropTypes from 'prop-types'
 import styled from 'styled-components'
 import Octicon from '@primer/octicons-react'
 import {COMMON, get} from './constants'
+import isNumeric from './utils/isNumeric'
 import theme from './theme'
 
-const sizeMapper = (size = 'medium') => {
-  if (typeof size === 'number') return size
-  const map = {
-    small: 56,
-    medium: 96,
-    large: 128
-  }
-  return map[size]
+const variantSizes = {
+  small: 56,
+  medium: 96,
+  large: 128
 }
 
-const sizeStyles = ({size}) => {
+const sizeStyles = ({size, variant}) => {
+  const calc = isNumeric(size) ? size : variantSizes[variant]
   return {
-    width: sizeMapper(size),
-    height: sizeMapper(size)
+    width: calc,
+    height: calc
   }
 }
 
 const CircleBadge = styled.div`
-  display: flex;
+  display: ${props => (props.inline ? 'inline-flex' : 'flex')};
```

Comment:
Curious why we decided to add the `inline` prop here?

Reply:
Oh, I added this when I was doing some testing and ended up leaving it. Happy to drop it!

Reply:
Let's keep it!